packages/wallets/src/features/cashier/screens/WithdrawalErrorScreen/WithdrawalErrorScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { TSocketError } from '@deriv/api-v2/types';
+import { WalletsErrorScreen } from '../../../../components';
+
+type TProps = {
+    error: TSocketError<'cashier'>['error'];
+    resetError: VoidFunction;
+    setResendEmail: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const WithdrawalErrorScreen: React.FC<TProps> = ({ error, resetError, setResendEmail }) => {
+    if (error.code === 'InvalidToken') {
+        return (
+            <WalletsErrorScreen
+                buttonText='Resend email'
+                buttonVariant='contained'
+                message='The verification link you used is invalid or expired. Please request for a new one.'
+                onClick={() => {
+                    resetError();
+                    setResendEmail(true);
+                }}
+                title='Email verification failed'
+            />
+        );
+    }
+
+    return (
+        <WalletsErrorScreen
+            buttonText='OK'
+            buttonVariant='contained'
+            message={error.message}
+            onClick={resetError}
+            title='Error'
+        />
+    );
```

Comment:
Wouldn't this return the wrong content for cryptocurrency maintenance:
<img width="1099" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/0c38c4c7-9422-4259-ac21-d9f583eaf4d8">

Reply:
NVM this will be covered in a separate card (https://app.clickup.com/t/20696747/WALL-4015) 🙏

Reply:
@lubega-deriv 
before refactoring i checked places where we use this component and seems like we only use it in withdrawal and deposit API errors, for locked scenarios we use `WalletsActionScreen` 🤔

Reply:
@heorhi-deriv this screen is for the scenario where the `CryptoConnectionError` API error is returned (for both deposit and withdrawal), but since a separate card was created for it we should handle it in a separate PR. Likely by adding a check for `error.code === 'CryptoConnectionError'` in this same component

Reply:
oh ok got it @lubega-deriv